scripts/test_updates_pg12.sh

```diff
@@ -1,25 +1,19 @@
 #!/usr/bin/env bash
 
 set -e
-set -o pipefail
 
 SCRIPT_DIR=$(dirname $0)
-echo $SCRIPT_DIR
 
-TAGS="1.7.0-pg12 1.7.1-pg12 1.7.2-pg12 1.7.3-pg12 1.7.4-pg12 1.7.5-pg12"
-TEST_VERSION="v6"
+source ${SCRIPT_DIR}/test_functions.inc
 
-TAGS=$TAGS TEST_VERSION=$TEST_VERSION bash ${SCRIPT_DIR}/test_updates.sh "$@"
-EXIT_CODE=$?
-if [ $EXIT_CODE -ne 0 ]; then
-    exit $EXIT_CODE
-fi
-
-TAGS="2.0.0-rc1-pg12 2.0.0-rc2-pg12 2.0.0-rc3-pg12 2.0.0-rc4-pg12 2.0.0-pg12 2.0.1-pg12 2.0.2-pg12 2.1.0-pg12"
-TEST_VERSION="v7"
-
-TAGS=$TAGS TEST_VERSION=$TEST_VERSION bash ${SCRIPT_DIR}/test_updates.sh "$@"
-EXIT_CODE=$?
-if [ $EXIT_CODE -ne 0 ]; then
-    exit $EXIT_CODE
-fi
+# There are repair steps between:
+#    1.7.1 and 1.7.2
+#    2.0.0-rc1 and 2.0.0-rc2
+#
+# Please extend this list if repairs are needed between more steps.
+run_tests -r -v6 \
+          1.7.0-pg12 1.7.1-pg12 1.7.2-pg12 1.7.3-pg12 1.7.4-pg12 1.7.5-pg12
+run_tests -r -v7 \
```

Comment:
Do we really care about testing rc candidates anymore? Wouldn't testing the 1.7 branch suffice here? Are any repair tests needed for 2.0.1 or 2.0.0?

Reply:
> Do we really care about testing rc candidates anymore?
This PR is just about factoring out the repair tests. I'm open for a discussion about what to test, but I do not think it should block this PR.
> Wouldn't testing the 1.7 branch suffice here?
Not sure I understand, isn't this what we are doing here?
> Are any repair tests needed for 2.0.1 or 2.0.0?
There are no repairs done when upgrading from 2.0.1 to 2.0.0 to later versions, so no tests are needed when updating from this version.